Makefile

```diff
@@ -53,10 +53,11 @@ flow:
 	./node_modules/.bin/flow check --strip-root
 
 lint:
-	./node_modules/.bin/eslint scripts $(SOURCES) *.js --format=codeframe --rulesdir="./scripts/eslint_rules"
+	./node_modules/.bin/eslint scripts $(SOURCES) '*.js' '**/.*.js' --format=codeframe --rulesdir="./scripts/eslint_rules"
 
 fix:
-	./node_modules/.bin/eslint scripts $(SOURCES) *.js --format=codeframe --fix --rulesdir="./scripts/eslint_rules"
+	./node_modules/.bin/prettier --config .prettierrc --write --ignore-path .eslintignore '**/*.json'
```

Comment:
Is the config file not found automatically?

Reply:
It is, but we have invalid package.json files in the repo which prettier wants to format and while searching for the config for this invalid file it searches in the file itself and then crashes because the json is invalid.

Reply:
Gotcha. Maybe worth a comment here so it is clear?

Reply:
added